packages/@react-spectrum/s2/intl/et-EE.json

```diff
@@ -1,18 +1,19 @@
 {
+  "breadcrumbs.more": "More items",
   "button.pending": "ootel",
   "contextualhelp.help": "Spikker",
   "contextualhelp.info": "Teave",
   "dialog.alert": "Teade",
   "dialog.dismiss": "Lõpeta",
   "dropzone.replaceMessage": "Kukuta fail asendamiseks",
   "inlinealert.informative": "Teave",
-  "inlinealert.negative": "Tõrge",
+  "inlinealert.negative": "Viga",
```

Comment:
Does this need to change in `@react-spectrum/inlinealert/intl/et-EE.json` too? This was originally copied from there

Reply:
Yes I'll change other occurrence as well, Viga is most used. I'll also do followup with vendor why we have both,